Backpack/Label/Classes/BPKLabel.m

```diff
@@ -100,11 +100,24 @@ - (void)setFontMapping:(BPKFontMapping *_Nullable)fontMapping {
     }
 }
 
+- (void)traitCollectionDidChange:(UITraitCollection *_Nullable)previousTraitCollection {
+    [super traitCollectionDidChange:previousTraitCollection];
+
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000
+    if (@available(iOS 13.0, *)) {
+        UITraitCollection *traitCollection = [UITraitCollection currentTraitCollection];
```

Comment:
We should use `self.traitCollection` instead since it's specific to the current instance.